cvat/apps/engine/serializers.py

```diff
@@ -809,6 +811,7 @@ class CloudStorageSerializer(serializers.ModelSerializer):
     key = serializers.CharField(max_length=20, allow_blank=True, required=False)
     secret_key = serializers.CharField(max_length=40, allow_blank=True, required=False)
     key_file_path = serializers.CharField(max_length=64, allow_blank=True, required=False)
```

Comment:
@Marishka17 , I believe key_file_path has to be removed. What do you think? Probably you can do that in a separate PR.

Reply:
Hmm, I added the option to upload a key file for a case when CVAT instance is deployed remotely. But now I've looked at it from a different side. Ok, I'll prepare PR for this.